flink-elastic-catalog/src/test/java/com/getindata/flink/connector/jdbc/catalog/ElasticCatalogTestBase.java

```diff
@@ -0,0 +1,112 @@
+package com.getindata.flink.connector.jdbc.catalog;
+
+import okhttp3.Credentials;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.apache.commons.compress.utils.IOUtils;
+import org.junit.BeforeClass;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.junit.Assert.assertTrue;
+
+class ElasticCatalogTestBase {
+    protected static ElasticJdbcCatalogFactory catalogFactory;
+    protected static String url;
+    protected static final ElasticsearchTestContainer container = new ElasticsearchTestContainer();
+    protected static final String USERNAME = "elastic";
+    protected static final String PASSWORD = "password";
+
+    private static boolean wasTrialEnabled = false;
+
+    @BeforeClass
+    public static void beforeAll() throws Exception {
+        container.withEnv("xpack.security.enabled", "true");
+        container.withEnv("ELASTIC_PASSWORD", PASSWORD);
+        container.withEnv("ES_JAVA_OPTS", "-Xms1g -Xmx1g");
+        container.start();
+        enableTrial();
+        catalogFactory = new ElasticJdbcCatalogFactory();
+
+        url = String.format("jdbc:elasticsearch://%s:%d", container.getHost(),
+                container.getElasticPort());
+    }
+
+    private static void enableTrial() throws Exception {
+        OkHttpClient client = new OkHttpClient();
+        Request request = new Request.Builder()
+                .url(String.format("http://%s:%d/_license/start_trial?acknowledge=true",
+                        container.getHost(), container.getElasticPort()))
+                .post(RequestBody.create(new byte[]{}))
+                .addHeader("Authorization", Credentials.basic(USERNAME, PASSWORD))
+                .build();
+        if (wasTrialEnabled) {
```

Comment:
Please do call to the elastic instead.

https://www.elastic.co/guide/en/elasticsearch/reference/current/get-license.html